### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."

### DISCUSSION THREAD

**Discussion by esQmo** [community/community#40118] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Unable to use Copilot anymore despite paying the fee. No suggestions showing. Had to delete the extensions, log out, change computer. Nothing seems to fix this issue! Where's the support? To who are we supposed to talk to???

**Comment by gunterh** [comment]
I'm having the same issue

**Comment by jeremiahsherrill** [comment]
shi% be broken is what's up.

**Comment by jfullstackdev** [comment, marked as answer]
I'm seeing a lot of these things, you can check this out since one GitHub Staff already responded there

https://github.com/orgs/community/discussions/40037
